.eslintrc

```diff
@@ -1,16 +1,29 @@
 ---
   extends: airbnb/legacy
+  env:
+    node: true
+    mocha: true
+    es6: true
```

Comment:
We aren't using any es6 features besides lambdas and retaining some basic Node.js 0.12 support is also good. Hence I would say let's please disable es6